src/Repository/CategoryRepository.php

```diff
@@ -19,6 +19,9 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Category::class);
     }
 
+    /**
+     * @return int|mixed|string
+     */
     public function findWithProducts()
```

Comment:
Why are you using this function? use findby. No?

Reply:
no